app/controllers/api/v1/search_results_controller.rb

```diff
@@ -44,6 +46,15 @@ def upload_form_params
       def render_success(serializer:, status:)
         render json: serializer, status: status
       end
+
+      def filter_url_equals(search_result_list)
+        url = params.dig(:filter, :url_equals)
+        if url
+          search_result_list.where('? = ANY (adwords_top_urls) OR ? = ANY (non_adwords_urls)', url, url)
+        else
+          search_result_list
```

Comment:
This code is not elegant 🙈 Since there will be more filters, it would be best to create a query object: https://github.com/nimblehq/travelbook-web/blob/development/app/queries/travelogues_query.rb